index.html

```diff
@@ -1989,7 +1989,7 @@
 <p>
   <span class="practicelab" id="principle-minimize-consent-requests">
     An [=actor=] should avoid interrupting a [=person=]'s use of a site for
-	  consent requests when an alternative is available.
+          consent requests when an alternative is available.
```

Comment:
I have emacs set to run whitespace-cleanup, so my bad.  I don't know how to revert this using the Web UI, which is all I have right now.